Contributing.md

```diff
@@ -0,0 +1,136 @@
+# Contributing to this repository
+The guidelines for contribution to babitz backend project are listed below. Please go through it before any contribution.
+
+## General guidelines
+- The general guidelines for writing code. We will be using a subset of [typescript contribution guidelines](https://github.com/microsoft/TypeScript/wiki/Coding-guidelines). In case any of these conflict with [loopback4 guidelines](https://loopback.io/doc/en/lb4/Best-practices.html), loopback4's guideline is taken as priority.
```

Comment:
```suggestion
The general guidelines for writing code. We will be using a subset of [typescript contribution guidelines](https://github.com/microsoft/TypeScript/wiki/Coding-guidelines). In case any of these conflict with [loopback4 guidelines](https://loopback.io/doc/en/lb4/Best-practices.html), loopback4's guideline is taken as priority.
```